storage/src/main/java/org/apache/kafka/storage/internals/log/LogValidator.java

```diff
@@ -63,6 +63,8 @@ public interface MetricsRecorder {
         void recordInvalidChecksums();
 
         void recordNoKeyCompactedTopic();
+        // Visible for test
+        Long meterCount(String metricName);
```

Comment:
we don't need to change the interface. you can have a specific impl which can return `meterCount`

Reply:
@chia7712 thanks for review.
I have updated this PR. PTAL.